# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1474231122 -7200
#      Sun Sep 18 22:38:42 2016 +0200
# Node ID 1872f49cfa2ee2b30e14e857262106e8b8c57bfc
# Parent  7ccbc2a6b99d8cb43b994a9b9226debff5273819
Use sombrero for the default image (bug #49009).

* octave-default-image.h: replace data indexes to produce a sombrero
instead of the Octave logo which can't be dithered correctly in the
"viridis" colormap.

diff --git a/libinterp/corefcn/octave-default-image.h b/libinterp/corefcn/octave-default-image.h
--- a/libinterp/corefcn/octave-default-image.h
+++ b/libinterp/corefcn/octave-default-image.h
@@ -22,268 +22,239 @@ along with Octave; see the file COPYING.
 
 #if ! defined (octave_octave_default_image_h)
 #define octave_octave_default_image_h 1
 
 #include "octave-config.h"
 
 static char default_im_data[] =
 {
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,20,20,20,20,20,20,21,21,21,21,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,20,20,20,20,20,20,20,20,20,20,20,21,21,
-  21,21,21,21,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,20,20,20,20,20,20,20,20,20,20,20,20,20,20,21,
-  21,21,21,21,21,21,21,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,20,
-  20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,
-  21,21,21,21,21,21,21,21,21,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,20,20,
-  20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,
-  20,21,21,21,21,21,21,21,21,21,21,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,20,20,20,20,
-  20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,20,
-  20,20,21,21,21,21,21,21,21,21,21,21,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,20,20,20,20,20,
-  20,20,20,20,20,20,20,20,20,20,20,20,31,31,31,31,
-  31,31,31,31,31,31,21,21,21,21,21,21,21,21,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,20,20,20,20,20,
-  20,20,20,20,20,20,20,20,20,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,21,21,21,21,21,21,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,20,20,20,20,20,20,
-  20,20,20,20,20,20,20,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,21,21,21,21,21,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,20,20,20,20,20,20,20,
-  20,20,20,20,20,20,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,21,21,21,
-  21,21,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,20,20,20,20,20,20,20,20,
-  20,20,20,20,20,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,63,
-  49,49,49,49,49,49,49,49,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,20,20,20,20,20,20,20,20,
-  20,20,20,20,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,49,
-  48,48,48,48,48,48,48,49,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,20,20,20,20,20,20,20,20,20,
-  20,20,20,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,49,
-  48,47,47,47,47,47,47,48,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,19,20,20,20,20,20,20,20,20,
-  20,20,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,49,
-  48,47,47,47,47,47,47,48,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,20,19,20,20,20,20,20,20,20,
-  20,20,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,49,
-  48,47,47,47,47,47,47,48,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,19,19,19,19,20,20,20,20,20,20,
-  20,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,49,
-  48,47,47,47,47,47,47,48,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,19,19,19,19,19,20,20,20,20,20,
-  20,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,49,
-  48,47,47,47,47,47,47,48,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,19,19,19,19,19,19,19,20,20,20,
-  19,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,49,
-  48,47,47,47,47,47,48,48,63,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,19,19,19,19,19,19,20,19,20,20,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,49,
-  49,48,48,48,48,48,48,48,21,31,31,31,31,31,31,31,
-  31,31,31,31,31,19,19,19,19,19,19,19,19,19,19,20,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,21,21,21,31,31,31,31,31,31,
-  31,31,31,31,31,20,20,20,20,20,20,20,20,20,20,20,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,21,21,31,31,31,31,31,31,
-  31,31,49,49,49,49,49,49,49,49,49,49,49,49,49,49,
-  49,49,49,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,21,21,31,31,31,31,31,
-  31,49,48,48,48,48,48,48,48,48,48,48,48,48,48,48,
-  48,48,48,49,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,21,21,21,31,31,31,31,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,21,21,31,31,31,31,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,21,21,31,31,31,31,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,21,21,31,31,31,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,21,21,31,31,31,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,21,21,21,31,31,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,21,21,31,31,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,21,21,31,31,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,21,21,21,31,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,21,21,31,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,21,21,31,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,20,21,31,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,21,21,21,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,20,21,21,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,20,20,21,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,20,20,20,
-  49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,20,20,20,
-  31,49,48,47,47,47,47,47,47,47,47,47,47,47,47,47,
-  47,47,48,49,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,20,20,20,
-  31,49,49,48,48,48,48,48,48,48,48,48,48,48,48,48,
-  48,48,49,63,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,20,20,20,
-  31,31,31,49,49,49,49,49,49,47,63,63,63,63,63,63,
-  63,63,20,19,19,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,20,20,20,20,
-  31,31,31,31,31,31,31,31,31,31,19,19,19,19,19,19,
-  19,19,19,19,19,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,20,20,20,20,
-  31,31,31,31,31,31,31,31,31,31,31,19,19,19,19,19,
-  19,19,19,19,19,19,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,20,20,20,20,
-  31,31,31,31,31,31,31,31,31,31,31,19,19,19,19,19,
-  19,19,19,19,19,19,19,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,20,20,20,20,
-  31,31,31,31,31,31,31,31,31,31,31,31,19,19,19,19,
-  19,19,19,19,19,19,19,19,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,20,20,20,20,20,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,19,19,19,
-  19,19,19,19,19,19,19,19,19,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,49,49,49,49,49,47,63,63,63,21,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,19,19,19,
-  19,19,19,19,19,19,19,19,19,19,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,49,49,48,48,48,48,48,48,48,48,48,48,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,19,19,
-  19,19,19,19,19,19,19,19,19,19,19,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,49,48,47,47,47,47,47,47,47,47,47,48,49,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,19,
-  19,19,19,19,19,19,19,19,19,19,19,19,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,49,47,47,47,47,47,47,47,47,47,47,48,48,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  19,19,19,19,19,19,19,19,19,19,19,19,19,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,49,47,47,47,47,47,47,47,47,47,47,48,49,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,19,19,19,19,19,19,19,19,19,19,19,19,19,19,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,49,47,47,47,47,47,47,47,47,47,47,48,49,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,19,19,19,19,19,19,19,19,19,19,19,19,19,19,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,49,47,47,47,47,47,47,47,47,47,47,48,49,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,19,19,19,19,19,19,19,19,19,19,19,19,19,
-  19,19,19,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,49,47,47,47,47,47,47,47,47,47,47,48,49,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,19,19,19,19,19,19,19,19,19,19,19,19,
-  19,19,19,19,19,31,31,31,31,31,31,31,31,31,31,31,
-  31,19,20,49,47,47,47,47,47,47,47,47,47,47,48,49,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,19,19,19,19,19,19,19,19,19,19,19,
-  19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,
-  19,20,20,49,47,47,47,47,47,47,47,47,47,47,48,49,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,18,19,19,19,19,19,19,19,19,19,
-  19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,
-  19,19,20,49,47,47,47,47,47,47,47,47,47,47,48,48,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,19,19,19,19,19,19,19,19,19,
-  19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,
-  19,20,20,49,48,47,47,47,47,47,47,47,47,47,48,49,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,19,19,19,19,19,19,19,
-  19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,
-  19,19,20,63,48,48,48,48,48,48,48,48,48,48,49,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,19,19,19,19,19,19,
-  19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,
-  19,19,19,20,21,63,63,63,48,49,49,49,49,49,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,19,19,19,19,
-  19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,
-  19,19,19,19,19,20,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,19,19,
-  19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,
-  19,19,19,19,19,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,19,19,19,19,19,19,19,19,19,19,19,19,19,19,19,
-  19,19,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,
-  31,31,31,31,31,19,19,19,19,19,19,19,19,19,19,31,
-  31,31,31,31,31,31,31,31,31,31,31,31,31,31,31,31
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,40,47,50,50,46,30,26,26,38,50,47,26,26,26,50,
+  46,26,50,46,26,26,26,50,46,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,43,55,54,50,51,55,
+  51,26,26,40,55,55,41,26,26,55,50,26,55,50,26,26,26,55,
+  50,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,30,54,52,30,26,26,46,55,43,26,40,55,55,52,30,
+  26,55,50,26,55,50,26,26,26,55,50,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,43,55,44,26,26,
+  26,26,26,26,26,40,55,47,55,46,26,55,50,26,55,50,26,26,
+  26,55,50,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,46,55,40,26,26,50,50,50,43,26,40,55,46,
+  46,54,34,55,50,26,55,50,26,26,26,55,50,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,44,55,43,
+  26,26,50,50,55,46,26,40,55,46,30,52,48,55,50,26,55,50,
+  26,26,26,55,50,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,38,55,50,26,26,26,40,55,46,26,40,
+  55,46,26,43,55,55,50,26,55,50,26,26,26,55,50,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  48,55,48,40,43,54,55,46,26,40,55,46,26,26,50,55,50,26,
+  52,55,43,40,47,55,46,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,30,48,55,55,55,47,48,46,
+  26,40,55,46,26,26,38,55,50,26,38,52,55,55,55,50,30,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,34,40,38,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,38,40,34,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,62,62,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  59,62,62,59,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,59,55,59,59,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,59,55,55,53,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,53,52,55,55,52,52,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,53,48,55,48,52,52,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,45,48,55,48,52,
+  53,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,44,48,48,48,52,45,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,44,44,48,44,
+  48,44,45,52,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,45,44,41,44,48,41,45,44,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,45,44,
+  37,44,37,41,45,44,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,37,41,37,44,37,41,36,36,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,44,
+  36,32,37,44,37,41,36,36,41,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,36,36,32,37,37,37,32,36,37,
+  41,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,20,
+  20,36,36,32,37,37,37,32,36,37,32,18,20,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,20,18,18,16,17,17,27,36,32,37,31,37,32,
+  36,37,32,17,17,18,18,18,20,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,20,18,18,18,16,16,
+  16,15,15,27,26,28,31,31,28,32,36,27,36,15,16,17,16,17,
+  17,18,18,20,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,18,20,17,17,16,16,16,16,15,15,32,27,24,28,24,31,
+  28,28,24,27,26,24,16,15,16,17,16,16,18,18,20,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,20,20,18,18,17,17,16,
+  16,14,14,15,24,27,24,28,24,31,28,28,24,27,26,28,12,14,
+  15,15,16,16,16,16,18,18,20,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,20,18,18,18,17,16,15,14,14,14,14,12,26,26,24,22,
+  24,24,28,22,24,26,26,22,12,12,12,12,14,14,16,16,16,18,
+  18,20,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,20,20,18,18,17,16,12,
+  15,12,10,10,10,12,26,18,24,22,24,24,24,22,24,26,26,22,
+  10,10,12,12,10,14,14,15,17,18,18,20,20,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,17,20,20,20,17,16,15,14,12,12,10,10,10,18,18,18,
+  16,17,24,24,24,22,16,18,18,24,18,6,9,10,10,14,14,15,
+  16,18,20,18,20,18,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,16,18,20,20,18,17,16,
+  15,10,10,7,6,6,6,15,18,18,15,17,18,24,17,22,16,18,
+  18,24,24,6,7,6,10,10,10,12,16,17,18,20,20,18,16,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,12,15,17,18,18,20,18,16,15,10,10,7,6,6,6,16,
+  18,16,15,17,18,18,17,22,16,18,18,16,17,5,4,6,6,10,
+  7,14,15,17,20,20,20,18,17,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,9,10,16,17,18,18,18,
+  18,15,10,10,6,5,4,4,4,10,12,16,15,17,12,18,17,12,
+  15,18,18,18,12,4,4,4,6,6,9,12,15,18,20,18,18,17,
+  16,12,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,9,9,12,15,16,17,18,18,20,15,10,10,4,5,4,4,
+  12,12,12,10,15,17,12,18,17,12,15,10,12,18,15,3,1,4,
+  4,4,5,10,16,20,18,18,17,17,16,12,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,7,7,9,9,12,15,26,26,
+  26,18,18,18,20,5,4,4,3,1,6,12,12,10,12,7,12,12,
+  12,12,9,10,12,12,9,12,1,1,4,4,6,20,18,18,18,17,
+  16,16,15,12,10,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,18,18,18,4,1,
+  1,1,9,12,12,9,12,7,9,12,12,12,9,10,12,12,9,12,
+  1,1,3,4,18,18,18,18,17,16,16,16,14,10,9,9,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,18,18,18,18,18,4,10,5,5,9,6,7,
+  9,7,12,6,9,10,5,5,10,6,5,18,18,18,18,18,26,26,
+  26,26,16,15,12,10,9,9,9,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,14,
+  17,17,16,18,18,20,18,20,6,7,9,9,7,6,9,5,20,18,
+  18,18,18,18,18,15,14,26,26,26,26,26,26,26,12,10,9,9,
+  7,7,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,10,16,16,16,16,17,18,17,
+  18,17,17,18,18,17,18,17,17,17,17,17,14,14,14,12,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,1,10,15,16,15,16,15,15,16,15,16,15,16,16,16,15,
+  16,16,17,14,14,10,12,1,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,1,10,15,15,14,
+  15,15,15,15,15,15,15,14,15,15,15,12,12,10,10,10,1,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,6,12,12,12,12,15,12,12,12,12,12,12,
+  12,15,10,9,7,6,6,1,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  12,12,12,12,12,12,12,12,12,10,12,7,10,7,6,1,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,10,10,12,10,10,10,10,
+  12,10,1,1,1,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,10,9,9,9,9,9,9,10,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,9,9,9,9,
+  9,9,9,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,7,7,7,7,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,40,48,50,50,44,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,43,55,54,50,52,55,
+  48,26,26,26,26,26,26,26,26,26,55,50,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,30,54,52,30,26,26,47,55,43,26,30,44,46,43,26,
+  26,38,55,51,34,26,34,46,46,43,26,43,46,26,26,40,46,34,
+  26,30,44,46,40,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,43,55,44,26,26,
+  26,34,55,48,30,51,55,50,55,48,26,46,55,54,43,34,55,54,
+  51,55,46,46,55,38,26,47,55,30,30,54,54,51,55,46,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,46,55,40,26,26,26,26,55,50,43,55,43,26,
+  46,55,34,26,55,50,26,43,50,34,26,51,50,34,55,46,26,52,
+  48,26,46,55,34,26,48,55,30,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,46,55,41,
+  26,26,26,26,55,50,46,55,30,26,26,26,26,26,55,50,26,26,
+  44,50,54,55,50,26,51,51,38,55,43,26,50,55,50,50,52,55,
+  40,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,38,55,48,26,26,26,43,55,46,46,55,
+  34,26,34,40,30,26,55,50,26,46,55,44,38,50,50,26,44,55,
+  46,55,30,26,50,55,40,40,40,40,30,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  50,55,47,40,43,54,54,34,41,55,47,30,48,55,30,26,55,50,
+  26,48,55,38,40,55,50,26,34,55,55,48,26,26,44,55,41,26,
+  46,48,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,30,50,55,55,55,52,38,26,
+  26,47,55,55,55,43,26,26,52,55,46,41,55,55,52,52,54,30,
+  26,50,55,43,26,26,30,50,55,55,54,43,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,34,40,40,26,26,26,26,26,38,40,34,26,26,26,
+  30,40,34,26,34,40,26,26,26,26,26,26,26,26,26,26,26,26,
+  38,40,30,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,26,
+  26,26,26,26,26,26,26,26,26,26
 };
 
 #endif
-
